tox.ini

```diff
@@ -6,7 +6,7 @@ toxworkdir = /tmp/.tox
 
 [testenv]
 deps = pytest
-       pytest-cookies
+       git+https://github.com/hackebrot/pytest-cookies.git@refs/pull/61/head
```

Comment:
did we want to switch to this branch globally or only for windows?

Reply:
Hmmm. Good question. I'm not sure. Maybe all tests, because it reflects the situation once--eventually--the upstream PR is merged? But I can see it either way.